### ISSUE TITLE
armv7 or armv6l?

### ISSUE THREAD

**Issue by vxgmichel** [show0k/docker-miniconda-armv7#1]
The architecture seems to be `armv6l`:
```console
$ docker run -it --rm show0k/miniconda-armv7 /bin/bash
root@8bf5a488e20d:/# uname -a
Linux 8bf5a488e20d 4.4.0-87-generic #110-Ubuntu SMP Tue Jul 18 12:55:35 UTC 2017 armv6l GNU/Linux
```
Am I missing something?

**Comment by show0k** [comment]
Yes, you're right, I mix up. It seem that the [base image is built upon](https://github.com/resin-io-library/resin-rpi-raspbian/blob/master/Dockerfile.no-systemd#L14) a ARMv6hf. I will update the README.
However, It was useful for me to compile for an ARMv7 because it was compatible.
